### DISCUSSION TITLE
How to setup slack integration for notification to upptime ?

### DISCUSSION THREAD

**Discussion by emothek** [upptime/upptime#633] [Q&A] [answered]
On github repo I added env variables to both environment secrets and Repository secrets as follows :
`NOTIFICATION_SLACK = true
NOTIFICATION_SLACK_STRATEGY = roundrobin
NOTIFICATION_SLACK_WEBHOOK = true
NOTIFICATION_SLACK_WEBHOOK_URL = XXXXXXXXXXXXX  `

but it doesn't trigger any notification action !

**Comment by AnandChowdhary** [comment, marked as answer]
Hmm, that's indeed not nice. Can you confirm that you've used your repository settings -> Secrets -> Actions and put them in there?

**Reply by emothek** [reply]
yes I confirm, there are two sections one named : Environment secrets     and    Repository secrets  
I tried them both

**Reply by rendiok** [reply]
I added the env variables as repository secrets and then ran the Uptime CI workflow in Actions. 

It works fine for me!
